service/src/main/java/bio/terra/landingzone/stairway/flight/create/AwaitCreateLandingResourcesZoneFlightStep.java

```diff
@@ -0,0 +1,74 @@
+package bio.terra.landingzone.stairway.flight.create;
+
+import bio.terra.landingzone.job.JobMapKeys;
+import bio.terra.landingzone.service.landingzone.azure.model.DeployedLandingZone;
+import bio.terra.landingzone.stairway.flight.exception.LandingZoneCreateException;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.FlightMap;
+import bio.terra.stairway.FlightState;
+import bio.terra.stairway.FlightStatus;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+import bio.terra.stairway.StepStatus;
+import bio.terra.stairway.exception.RetryException;
+import java.util.Optional;
+
+public class AwaitCreateLandingResourcesZoneFlightStep implements Step {
+  public static final int FLIGHT_POLL_SECONDS = 1;
```

Comment:
Given the length of time this takes, should we poll less frequently than every second?

Reply:
I think it make sense.